docs/supervisor-notes.md

```diff
@@ -16,6 +16,8 @@ Below is the networking diagram of velero-plugin-for-vsphere in vSphere with Tan
 
 Certain resources with the **vSphere with Tanzu** Supervisor Cluster need to be created by the Supervisor Cluster and cannot be restored. The **Velero Plugin for vSphere** blocks backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as "Partially Failed".
 
+Similarly, certain resources with the **Tanzu Kubernetes Grid** multicloud (TKGm) need to be created by the TKGm cluster and cannot be restored. The **Velero Plugin for vSphere** blocks backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as "Partially Failed".
```

Comment:
What are the resources that cannot be restored?  Are you saying the backup is successful but restore failed?

Reply:
No. Both backup and restore will fail. In general, during backup and restore, if user does not specify "--exclude-resources"  to exclude the resource listed in the resource block list, backup/restore will failed with error like "Resource CRD XXX is blocked in backup, skipping".
We have already documented this for Supervisor cluster(line 16-18), and we observed the same behavior in TKGm, and that is why we added line 19 for TKGm.